components/prism-rescript.js

```diff
@@ -0,0 +1,54 @@
+Prism.languages.rescript = {
+	'comment': {
+		pattern: /\/\/.*|\/\*[\s\S]*?(?:\*\/|$)/,
+		greedy: true
+	},
+	'string': {
+		pattern: /"(?:\\(?:\r\n|[\s\S])|[^\\\r\n"])*"/,
+		greedy: true
+	},
+	'class-name': /(\b[A-Z]\w*)|(@[a-z.]*)|#[A-Za-z]\w*|#\d/,
```

Comment:
The capturing groups here are why the tests fail. Please only use capturing group in combination with backreferences or `lookbehind: true`.
```suggestion
	'class-name': /\b[A-Z]\w*|@[a-z.]*|#[A-Za-z]\w*|#\d/,
```

Reply:
Done